exonum/src/runtime/rust.rs

```diff
@@ -40,14 +39,14 @@ struct RustRuntimeInner {
     initialized: HashMap<ServiceInstanceId, RustArtifactData>,
 }
 
-#[derive(Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct RustArtifactSpec {
     name: String,
-    version: (usize, usize, usize),
+    version: (u32, u32, u32),
```

Comment:
Is it temporary solution, isn't it ? There is a `semver` create exists for such goal.

Reply:
Yep, it is. I think it will be rewritten (almost) completely when we'll work on `RustRuntime`. For now it's important only as an interface part.